### PR TITLE
Add Linux information to CEF article

### PR DESCRIPTION
**Pull request by ZackaryH8**

### CODE REVIEW

**Comment by ZackaryH8** [comment]
Might be irrelevant if the actual issue is fixed [here](https://github.com/altmp/altv-issues/issues/1412)

**Review by Lhoerion — APPROVED** [review]
Nicely done :)

**Comment by lolsniperftw** [comment]
Note that even if the issue is fixed by the devs then only the part with copying files becomes irrelevent.
The part about running the game using Windows of Steam, etc ,,, will still be relevant.

**Comment by Lhoerion** [comment]
@ZackaryH8 Inner talks had taken place about where to put this section and we came to the conclusion that Troubleshooting article seems like a more appropriate spot. However, since CEF article is actually connected to this particular remark, I'd recommend to leave a note there with a proper reference link

**Review by Lhoerion — CHANGES_REQUESTED** [review]
@ZackaryH8 Inner talks had taken place about where to put this section and we came to the conclusion that Troubleshooting article seems like a more appropriate spot. However, since CEF article is actually connected to this particular remark, I'd recommend to leave a note there with a proper reference link

**Comment by ZackaryH8** [comment]
So to clarify I will move this information to troubleshooting and add a link to cef page to the troubleshooting page?

**Comment by Lhoerion** [comment]
First part correct, second one more like "add a link in cef page to the troubleshooting page"

**Comment by ZackaryH8** [comment]
Let me know if you need to do anything else

File: articles/troubleshooting/client.md [hunk 1]
```diff
@@ -82,6 +82,19 @@ If you are stuck under the world after launching alt:V or see a grey world (<a h
 4. Add "altv.exe", "altv-webengine.exe"and "gta5.exe" and select "High Performance"
 5. More detailed description can be found <a href="https://www.amd.com/en/support/kb/faq/dh-017">here</a>
 
+###### Linux
```
Comment (Lhoerion) [inline articles/troubleshooting/client.md:85]: Change to `h4`, so it'll be marked for anchoring. Also title more like `Webview not rendering on Linux`. Then you can link it like `./troubleshooting/client.md#webview-not-rendering-on-linux`

**Review by Lhoerion — CHANGES_REQUESTED** [review]

**Review by Unnvaldr — APPROVED** [review]

**Pull request merged**